src/cr/cube/crunch_cube.py

```diff
@@ -940,6 +940,19 @@ def _calculate_std_res(self, counts, total, colsum, rowsum, slice_):
     def _col_direction_axis(self):
         return self.ndim - 2
 
+    def _counts(self, weighted):
+        """Return _BaseMeasure subclass for *weighted* counts.
+
+        The return value is a _WeightedCountMeasure object if *weighted* is
+        True and the cube response is weighted. Otherwise it is an
+        _UnweightedCountMeasure object. Any means measure that may be present
+        is not considered. Contrast with `._measure()` below.
+        """
+        return (
+            self._measures.weighted_counts if weighted else
+            self._measures.unweighted_counts
+        )
```

Comment:
Writing it like this:
```
if weighted:
    return self._measures.weighted_counts
return self._measures.unweighted_counts
```
Increases the unit test coverage from 65.94 to 66.01 percent.